src/webots/control/WbLanguageTools.cpp

```diff
@@ -63,7 +63,11 @@ const QStringList WbLanguageTools::javaArguments() {
 
 QString WbLanguageTools::pythonCommand(QString &shortVersion, const QString &command, QProcessEnvironment &env) {
   QString pythonCommand = command;
-  const QString advice =
+  QString advice =
```

Comment:
Can't we keep it const?
```suggestion
  const QString advice =
```